ccd-definition/CaseEventToComplexTypes/AddOrAmendClaimDocuments.json

```diff
@@ -13,7 +13,7 @@
     "ID": "Upload",
     "CaseEventID": "ADD_OR_AMEND_CLAIM_DOCUMENTS",
     "CaseFieldID": "servedDocumentFiles",
-    "ListElementCode": "particularsOfClaimDocument",
+    "ListElementCode": "particularsOfClaimFile",
```

Comment:
I don't think this field rename was necessary

Reply:
I thought it would make sense to keep the names consistent when referring to that element